gips/0081-indexing-payments.md

```diff
@@ -0,0 +1,165 @@
+---
+GIP: 0081
+Title: A New Proposal for Indexing Payments
+Authors: Matias <matias@edgeandnode.com>, Pablo <pablo@edgeandnode.com>
+Created: 2024-10-21
+Updated: 2024-10-21
+Stage: Draft
+Discussions-To: <TODO>
+Category: Protocol Logic
+Depends-On: GIP-0066, GIP-0068
+Replaces: GIP-0058
+---
+
+## Abstract
+
+This GIP describes **Indexing Payments**, a mechanism by which a protocol participant (commonly a gateway) can incentivize a candidate indexer to service a subgraph. It details a target, trust-minimized end-state that relies on Graph Horizon, while also introducing a semi-trusted Minimum Viable Product (MVP) based on the current protocol and off-chain components (gateway + indexer stacks). This GIP is an alternative, simplified proposal to [GIP-0058](https://github.com/graphprotocol/graph-improvement-proposals/blob/main/gips/0058-Replacing-Bonding-Curves-with-Indexing-Fees.md).
+
+## Motivation
+
+Curation has generally been perceived as a source of friction for Developers that may only want to have subgraphs indexed to power their decentralized applications. It requires locking up certain amounts of GRT, and the necessary amounts might vary depending on curation on other subgraphs. While the quality of service provided on the network is generally high, Indexers collect rewards for posting Proofs of Indexing independently to the query service they provide, so Developers are not guaranteed a certain level of service for the amount of curation they put into the system.
```

Comment:
I am not sure what the first sentance means. Are you trying to say that Curation does not provide a convenient way for Developers to have the subgraphs they need indexed? (Perhaps say that more directly?)

From second second sentance I would remove "up certain amounts of", for "the necessary amounts might vary ..." you might want to consider simply saying the amount required is not always clear and is influenced by other activity on the network?

Reply:
re-worded